src/index.test.ts

```diff
@@ -21,45 +21,47 @@
 
 import { describe, it, expect } from "@jest/globals";
 import {
+  AccessGrantWrapper,
   approveAccessRequest,
   cancelAccessRequest,
+  createContainerInContainer,
   denyAccessRequest,
+  fetchWithVc,
+  gConsent,
+  getAccessApiEndpoint,
   getAccessGrant,
   getAccessGrantAll,
-  getAccessApiEndpoint,
   getAccessGrantFromRedirectUrl,
   getAccessManagementUi,
-  getAccessRequestFromRedirectUrl,
-  isValidAccessGrant,
-  issueAccessRequest,
-  redirectToAccessManagementUi,
-  redirectToRequestor,
-  revokeAccessGrant,
-  fetchWithVc,
-  getFile,
-  saveFileInContainer,
-  overwriteFile,
-  getSolidDataset,
-  saveSolidDatasetAt,
-  GRANT_VC_URL_PARAM_NAME,
-  gConsent,
-  odrl,
-  AccessGrantWrapper,
   getAccessModes,
+  getAccessRequestFromRedirectUrl,
   getExpirationDate,
+  getFile,
   getId,
   getIssuanceDate,
   getIssuer,
   getRequestor,
   getResourceOwner,
   getResources,
+  getSolidDataset,
   getTypes,
+  GRANT_VC_URL_PARAM_NAME,
+  issueAccessRequest,
+  isValidAccessGrant,
+  odrl,
+  overwriteFile,
+  redirectToAccessManagementUi,
+  redirectToRequestor,
+  revokeAccessGrant,
+  saveFileInContainer,
+  saveSolidDatasetAt,
 } from "./index";
```

Comment:
These imports aren't actually intended to be alphabetically sorted, instead they're more grouped by the functionality / section of the SDK — it's a little confusing, but it basically tries to keep the various parts of the SDK together as a block